LanguageFeatures/primary-constructors/static_processing_A05_t02.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A compile-time error occurs if the name of the primary
+/// constructor is the same as the name of a constructor (declaring or not) in
+/// the body.
+///
+/// @description Check that it is a compile-time error if the name of the
+/// primary constructor is the same as the name of some constructor declared in
+/// the body. Test extension types.
+/// @author sgrekhov22@gmail.com
+
+// TODO (sgrekhov) Add `declaring-constructors` experimental flag
+
+extension type ET1(int v) {
+  ET1(this.v);
+//^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+extension type ET2 {
+  this(int v);
+  ET2(int v);
```

Comment:
This one would have a spurious error because it never initializes the representation variable, so we might have a bug about name clashes and still succeed.

Reply:
Indeed. Changed to `ET2(this.v);`.